### PR TITLE
bump xrootd to rc2 and add python support

### PR DESCRIPTION
**Pull request by davidlange6**
this time cherry-picking.
the python changes are not tested so believing davida's suggestions..

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: xrootd.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### RPM external xrootd 4.5.0
 ## INITENV +PATH LD_LIBRARY_PATH %i/lib64
-%define tag 6a5db61a4ef13d1e79da71af47fbb8a660cf8aac
+%define tag af6aebbbbe7da7fd89f3698c1e485a79a5990037
```
Comment (davidlt) [inline xrootd.spec]: Why is this commit needed? Looks to be useless here.

Comment (davidlange6) [reply xrootd.spec]: i was following the commit history from the gcc7 branch.. (unneeded I guess)

Comment (davidlt) [reply xrootd.spec]: I don't see such commit in the history: https://github.com/cms-sw/cmsdist/commits/IB/CMSSW_9_1_X/gcc700/xrootd.spec

**Comment by davidlange6** [comment]
Ah- then I first reverted the commit that added the previously proposed fix to the gcc530 branch to put the branches back in sync.

David


On 21 Apr 2017, at 19:09, davidlt <notifications@github.com<mailto:notifications@github.com>> wrote:


@davidlt commented on this pull request.

________________________________

In xrootd.spec<https://github.com/cms-sw/cmsdist/pull/2975#discussion_r112735069>:

> @@ -1,6 +1,6 @@
 ### RPM external xrootd 4.5.0
 ## INITENV +PATH LD_LIBRARY_PATH %i/lib64
-%define tag 6a5db61a4ef13d1e79da71af47fbb8a660cf8aac
+%define tag af6aebbbbe7da7fd89f3698c1e485a79a5990037


I don't see such commit in the history: https://github.com/cms-sw/cmsdist/commits/IB/CMSSW_9_1_X/gcc700/xrootd.spec

**Pull request merged**